### ISSUE TITLE
1.38.1 - ERROR in multi manager

### ISSUE THREAD

**Issue by PEM--** [kadirahq/react-storybook#318]
Since the upgrade in 1.38.1, I get this error:

```
ERROR in multi manager
Module not found: Error: Cannot resolve 'file' or 'directory' /Users/PEM/Projects/panoply/app/node_modules/@kadira/storybook/dist/manager in /Users/PEM/Projects/panoply/app
 @ multi manager
```

Downgrading to 1.38.0 fixes the error.

**Comment by mnmtanish** [comment]
@PEM-- thanks for reporting. I'll check it out right away.

**Comment by mnmtanish** [comment]
Released `v1.38.2` with a fix

**Comment by PEM--** [comment]
Thanks. It works like a charm. By the way, thanks also for the addons. `react-storybook-addon-info` is a very neat idea 👍
